compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -665,6 +648,60 @@ template <typename Config> class MapAllocator {
   LocalStats Stats GUARDED_BY(Mutex);
 };
 
+template <typename Config>
+void *MapAllocator<Config>::tryAllocateFromCache(
+    const Options &Options, uptr Size, uptr Alignment, uptr HeadersSize,
```

Comment:
Do we need `HeaderSize`? I think we can call `getHeaderSize()` in the function.

Reply:
I wanted to keep the header size in `tryAllocateFromCache()` consistent with the header size in `allocate()`. But since only `allocate()` calls `tryAllocateFromCache()`, I guess it's not necessary.

Reply:
Right, getHeaderSize() should be used everywhere in the secondary allocator. If there's any inconsistency then it's better not to be a function (or has to have a different name at least)